learn/sherif/main_test/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -26,26 +28,37 @@
  */
 public class RobotContainer {
     // The robot's subsystems and commands are defined here...
-    private Factory m_f = new Factory();
-    private DriveTrain m_drivetrain;
-    private Joystick m_controller;
+    public Factory factory = new Factory();
+    private DriveTrain m_drivetrain = null;
+    private Joystick m_controller = null;
+    private IntakeControl m_intake = null;
```

Comment:
Wouldn't you be fine just doing `private DriveTrain m_drivetrain;`? I don't see why the null needs to be explicit, Java does that by default.``